<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentTab</class>
 <widget class="QWidget" name="ContentTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxBlockPopups">
     <property name="text">
      <string>Block pop-up windows</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAutoLoadImages">
     <property name="text">
      <string>Load images automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxEnableJavaScript">
     <property name="text">
      <string>Enable JavaScript</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFonts">
     <property name="title">
      <string>Fonts</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelDefaultFont">
        <property name="text">
         <string>Default Font:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fontComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="labelFontSize">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFontsAdvanced">
        <property name="text">
         <string>Advanced</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>300</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
